src/lib/site-import/status.ts

```diff
@@ -276,9 +304,10 @@ ${ maybeExitPrompt }
 							jobStatus = 'error';
 						} else if ( statusSteps.every( ( { result } ) => result === 'success' ) ) {
 							jobStatus = 'success';
-							importJob.completedAt = new Date(
-								Math.max( ...statusSteps.map( step => step.finished_at ), 0 ) * 1000
-							).toUTCString();
+							const timestamps = statusSteps
+								.map( step => step.finished_at )
+								.filter( Boolean ) as number[];
+							importJob.completedAt = new Date( Math.max( ...timestamps, 0 ) * 1000 ).toUTCString();
```

Comment:
`Math.max()`, according to TS, does not like all kinds of `null` and `undefined`.